schema/timestamptz.go

```diff
@@ -41,7 +41,14 @@ func (dst *Timestamptz) Equal(src CQType) bool {
 	}
 
 	if value, ok := src.(*Timestamptz); ok {
-		return dst.Status == value.Status && dst.Time.Equal(value.Time) && dst.InfinityModifier == value.InfinityModifier
+		if dst.Status != value.Status || dst.InfinityModifier != value.InfinityModifier {
+			return false
+		}
+
+		if dst.Time.Sub(value.Time) < 1*time.Second {
```

Comment:
Oof, this might bite us later. Why is it necessary to round to 1 second?

Reply:
yeah. ok I think I work around that with rounding up our auto-generated data in the tests to seconds. I only hit this snag in csvs.